### PR TITLE
13.0 mig project parent task filter

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@astirpe @Saran440 @schout-it and @nikul-serpentcs could you review this PR?

**Review by astirpe — APPROVED** [review]
Tested in Runbot and it works fine!

Just one think, not blocking: the filters for parent_id should be visible only if user belongs to group "Use Subtasks".

**Comment by emagdalenaC2i** [comment]
@astirpe Thanks, I will take a look to add that option later

**Review by schout-it — APPROVED** [review]
looks good, but in a future iteration look at the filters as @astirpe said.
showing filter options for something a user can't see on the views is just strange

File: project_parent_task_filter/views/project_task.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+             <field name="stage_id" position="after">
+                <field name="parent_id" string="Parent task"/>
+            </field>
+
+            <xpath expr="//filter[@name='rating_task']" position="after">
+                <filter string="Parent tasks" name="parent_only" domain="[('parent_id','=',False)]"/>
+            </xpath>
+
+            <xpath expr="//filter[@name='company']" position="after">
+                <filter name="parent_task" string="Parent tasks" domain="[]" context="{'group_by' : 'parent_id'}"/>
```
Comment (Saran440) [inline project_parent_task_filter/views/project_task.xml]:
```suggestion
                <filter name="parent_task" string="Parent tasks" context="{'group_by' : 'parent_id'}"/>
```
You can delete domain function, If you don't used it.

**Review by Saran440 — APPROVED** [review]
Functional test. :+1:

**Comment by emagdalenaC2i** [comment]
I don't know why now but travis says no...
Seems something about pre-commit
image
I will take a look later

**Pull request closed without merging**